tests/unit/backend/utils/field-validation/phone-num-validation.spec.js

```diff
@@ -2,7 +2,7 @@ const {
   validateField,
 } = require('../../../../../dist/backend/app/utils/field-validation')
 
-describe('Mobile validation tests', () => {
+describe('Phone number validation tests', () => {
```

Comment:
Can we separate the tests for mobile from the tests for homeno into separate files?